### DISCUSSION TITLE
de/1-07-prompts-erstellen/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#58] [General]
# de/1-07-prompts-erstellen/

https://ai.lernos.org/de/1-07-prompts-erstellen/

**Comment by KathaZ** [comment]
A2/Z2: KI-Systemen, (Komma weg)
A2/Z2: ein*:e* Benutzer*:in*
A2/Z5,6: vo*n den* Nutzer*:innen*
A3/Z7: eignet*."*
A3/Z14: das *G*anze 
A4/Z9: Text*,* der 
A5/Z11: sind*,* neigen
A5/Z16: Code-Interpreter)*,*
A6/Z3: *die* Nutzer*:innen erwarten.*
A6/Z4: One-Shot*-*Learning
A6/Z6: One-Shot*-*Prompting
A6/Z7: Few-Shot*-*Learning
A6/Z11: Few-Shot*-*Prompting:
A10/Z5: *b*ildgenerierenden KIs 
Generell: *z. B.* mit kleinem festen Leerabstand, nicht kompress gesetzt
